docker-java-api/src/main/java/com/github/dockerjava/api/model/Bind.java

```diff
@@ -99,36 +99,39 @@ public static Bind parse(String serialized) {
         try {
             String[] parts = serialized.split(":");
             switch (parts.length) {
-            case 2: {
-                return new Bind(parts[0], new Volume(parts[1]));
-            }
-            case 3: {
-                String[] flags = parts[2].split(",");
-                AccessMode accessMode = AccessMode.DEFAULT;
-                SELContext seMode = SELContext.DEFAULT;
-                Boolean nocopy = null;
-                PropagationMode propagationMode = PropagationMode.DEFAULT_MODE;
-                for (String p : flags) {
-                    if (p.length() == 2) {
-                        accessMode = AccessMode.valueOf(p.toLowerCase());
-                    } else if ("nocopy".equals(p)) {
-                        nocopy = true;
-                    } else if (PropagationMode.SHARED.toString().equals(p)) {
-                        propagationMode = PropagationMode.SHARED;
-                    } else if (PropagationMode.SLAVE.toString().equals(p)) {
-                        propagationMode = PropagationMode.SLAVE;
-                    } else if (PropagationMode.PRIVATE.toString().equals(p)) {
-                        propagationMode = PropagationMode.PRIVATE;
-                    } else {
-                        seMode = SELContext.fromString(p);
-                    }
+                case 2: {
+                    return new Bind(parts[0], new Volume(parts[1]));
                 }
+                case 4: {
```

Comment:
please put it after the 3rd case

Reply:
Presumably it's handled here to get "fall through" behaviour after consolidating the first two parts. Would it be better to move this to an `if (parts.length == 4)` block before the `switch`?

Also, does Docker for Windows ever present a variant without the access mode parts (i.e. is there a 3-part version like `C:/foo:/bar`) that `case 3` wouldn't handle?